### ISSUE TITLE
ShellHost.exe -System Error

### ISSUE THREAD

**Issue by Kawika498** [simeononsecurity/Windows-Optimize-Harden-Debloat#103]
### Discussed in https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/discussions/102

<div type='discussions-op-text'>

<sup>Originally posted by **Kawika498** November 27, 2024</sup>
I did a fresh install of Windows on this Desktop I am hardening After running the script and reseating the device when I log into my Admin user the desktop keeps crashing and a prompt saying that the system detected an overrun of stack-based buffer in this application. Are there any idea on what might be causing this I have done 3 fresh installs just to make sure it wasn't an application that was causing the issue</div>

**Comment by Kawika498** [comment]
I am using Windows 11 24H2 and I read the readme file saying to create a local user. However I am confused on if I need to create another local user after the initial setup.

**Comment by simeononsecurity** [comment]
> I am using Windows 11 24H2 and I read the readme file saying to create a local user. However I am confused on if I need to create another local user after the initial setup.

You should only have to create local accounts in lue of any accounts originally set up with a Microsoft login with email.

**Comment by simeononsecurity** [comment]
> ### Discussed in #102
> 
> Originally posted by **Kawika498** November 27, 2024 I did a fresh install of Windows on this Desktop I am hardening After running the script and reseating the device when I log into my Admin user the desktop keeps crashing and a prompt saying that the system detected an overrun of stack-based buffer in this application. Are there any idea on what might be causing this I have done 3 fresh installs just to make sure it wasn't an application that was causing the issue

As far as your issue goes, we've not been able to replicate the issue on W11 24H2 base with all updates installed.

We need you to answer all of the questions the bot has asked to continue.

- The specific Windows version you're on. You can get this information by running `winver` in the Run dialog or by checking the System Information. 
- The commands and examples you used. 
- The version of the script that was run and from where. 
- If using the manual option, the configuration flags that were used.

**Comment by GreenCyborgNinja** [comment]
I ran into the same issue. First the crashing as well on Desktop, neither safe mode or anything could fix it. I needed to reinstall Windows again. One new Windows 11 install from UDUMP + Rufus later I installed the usual software I use (DC,Steam,LibreWolf), updated Drivers and then ran the script. After a restart the ShellHost.exe - System Error message came up again, but without crashing my desktop. Current version: 24H2 (OS Build 26100.3323) W11 Pro
